.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@4.14.1
```

Comment:
This bumps apptestctl with the updated AppCatalogEntry CRD.

There is also a fix for this in https://github.com/giantswarm/app-operator/pull/961